css/main.css

```diff
@@ -495,10 +582,23 @@ footer {
   }
 
   .features-content {
-    padding: 0px 250px 80px 210px;
+    padding: 0px 110px 80px 110px;
   }
 }
 
+@media screen and (max-width: 1240px) {
+  .cards {
+    grid-template-columns: repeat(1, 2fr);
+    max-width: 960px;
+    margin: auto;
+  }
+  .card:hover {
+    transform: scale(1);
+  }
+  .features-content {
+    padding: 0px;
+  }
+}
```

Comment:
Can we please move this code to already defined media queries ??
Check if 1240px is already defined or not

Reply:
There was no media query for 1240px that's why created one.